src/tapeimage.cpp

```diff
@@ -660,9 +660,8 @@ void tapeimage::seek(std::int64_t n) noexcept (false) {
         const auto real_offset = this->addr.physical(n, pos);
 
         /*
-         * If n = 0 there's no previous, and to remain consistent with open(),
-         * the read head is *not* put at the first byte of the preceeding
-         * header. This is effectively the same as any other position.
+         * To remain consistent with open(), the read head is *not* put at the
```

Comment:
Refers to code removed several commits after, so feel free to disregard.

1) No information that this is valid for n=0 only.
2) maybe
`put at the (last) byte of preceding header`
first byte of preceding header makes not much sense
3) `preceeding` apparently is wrong spelling for the local variable :confused: